tests/bud.bats

```diff
@@ -1046,8 +1046,10 @@ load helpers
 
 @test "bud with unused build arg" {
   target=busybox-image
-  out=$(buildah bud --signature-policy ${TESTSDIR}/policy.json -t ${target} --build-arg foo=bar --build-arg foo2=bar2 -f ${TESTSDIR}/bud/build-arg ${TESTSDIR}/bud/build-arg | grep "Warning" | wc -l)
-  [ "$out" -ne 0 ]
+  run_buildah bud --signature-policy ${TESTSDIR}/policy.json -t ${target} --build-arg foo=bar --build-arg foo2=bar2 -f ${TESTSDIR}/bud/build-arg ${TESTSDIR}/bud/build-arg
+  expect_output --substring "one or more build args were not consumed: \[foo2\]"
+  run_buildah bud --signature-policy ${TESTSDIR}/policy.json -t ${target} --build-arg IMAGE=busybox -f ${TESTSDIR}/bud/build-arg/Dockerfile2 ${TESTSDIR}/bud/build-arg
+  ! expect_output --substring "one or more build args were not consumed: \[IMAGE\]"
```

Comment:
Not really necessary for this PR, but might be nice to change to "IMAGE=alpine" and then test that the alpine image did get made "exepect_output --substring "STEP 1: FROM alpine"  and "[ "$status" -eq 0 ]"

Reply:
Updated to add two of those - the `run_buildah` function checks the exit status for us.